integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/resources/SlackResource.java

```diff
@@ -25,7 +26,7 @@ public static List<String> getListOfSlackMessages() {
                 .oauth2(SLACK_TOKEN)
                 .contentType(ContentType.JSON)
                 .queryParam("channel", SLACK_CHANNEL)
-                .queryParam("oldest", oneHourAgo.toEpochSecond(ZonedDateTime.now().getOffset()))
+                .queryParam("oldest", oneHourAgo.toEpochSecond(ZonedDateTime.now(ZoneOffset.UTC).getOffset()))
```

Comment:
The previous implementation was intentional to retrieve local offset. The change breaks it for zones > UTC.

Reply:
Above code is failing in local

Reply:
Could you try again now? Thanks

Reply:
Looks good!